pibooth_picture_template.py

```diff
@@ -395,13 +403,13 @@ def _build_outlines(self, image):
         :type image: :py:class:`PIL.Image`
         """
         draw = ImageDraw.Draw(image)
-        for pos_x, pos_y, width, height, angle in self._iter_images_rects():
+        for pos_x, pos_y, width, height, angle,value in self._iter_images_rects():
```

Comment:
I don't understand why did you add the value if you don't use it?

Reply:
I use it above ( line 354), here I added it because otherwise it throws an error.